### DISCUSSION TITLE
Quick access to feedback throughout Android app

### DISCUSSION THREAD

**Discussion by wesort** [community/community#49498] [Mobile]
As I navigate around the app it would be great to search for and report bugs & FRs while staying in context. An ever-present link to: Profile > Settings > Share feedback 

I'm using the GitHub Android app Version 1.102.0 on a Pixel 6

**Comment by 0xNevo** [comment]
To enable an ever-present link to share feedback in the GitHub Android app, we can do the following:

Open the GitHub Android app on your Pixel 6.
Tap on your profile icon (located at the bottom right corner of the screen).
Select "Settings" from the drop-down menu.
Go to the "Advanced" section and tap on "Enable debug logging".
Once debug logging is enabled, go back to the home screen of the GitHub app and navigate to the area where you want to report a bug or FR.
At this point, you will now see a feedback icon on the top right corner of the screen that looks like a chat bubble with a plus sign inside it.
Tapping on the feedback icon will open a form where you can enter details about the bug or FR you want to report. Make sure to fill out all relevant information so the development team can understand the issue properly.
Once you submit your feedback, it will be sent to the GitHub team for review and action.
By enabling debug logging and accessing the feedback feature via the chat bubble, you can quickly report any issues you encounter while using the app without having to leave the context of the screen you're on.

**Reply by wesort** [reply]
That sounds great, but this bit doesn't appear for me:
> Go to the "Advanced" section and tap on "Enable debug logging".

![image](https://user-images.githubusercontent.com/3920126/223722235-2d99f802-5a14-4057-966e-6da32d0df2aa.png)![image](https://user-images.githubusercontent.com/3920126/223722186-d17e65e3-698d-423d-8633-22cd2864249b.png)

**Comment by jsoref** [comment]
I just filed https://github.com/orgs/community/discussions/49597 which notes that even the feedback thing is at least one additional tap too many. It should really take you straight to the new mobile category discussion...
